File: redditmemes/app/src/main/java/com/example/redditmemes/MainActivity.java
package com.example.redditmemes;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

import com.example.redditmemes.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

     ActivityMainBinding binding;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        setContentView(R.layout.activity_main);
        
        binding.next.setOnDragListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                getMeme();
            }
        });
        
        binding.share.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                shareMeme();
            }
        });
        
    }

    private void getMeme() {
    }

    private void shareMeme() {
        
    }
}

File: PortfolioApp/app/src/main/java/com/example/portfolioapp/MainActivity.kt
package com.example.portfolioapp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val  buttonskils = findViewById<Button>(R.id.btnskills)

        buttonskils.setOnClickListener {
            intent = Intent(this,SkillsActivity::class.java)
            startActivity(intent)
        }
    }
}